app/views/content/is-teaching-right-for-me/teacher-training-subjects.md

```diff
@@ -0,0 +1,297 @@
+---
```

Comment:
@caitlin-reidy @martyn-w Think this file has snuck onto this PR erroneously and needs removing?

Reply:
Once this is done, it'll clear up the build failures :-)

Reply:
Hi @caitlin-reidy, @gemmadallmandfe, the file `app/views/content/is-teaching-right-for-me/teacher-training-subjects.md` was added on 24/1/2024 and since then has been updated a further 6 times.

I don't think we can revert these commits without loosing other work. If this file does not belong to this PR, please take a copy of it and then delete it.